env.dist

```diff
@@ -54,6 +54,9 @@ export LOGIN="http://localhost:3000"
 # What is our fully qualified login API acces point?
 export LOGINAPI="http://testuser:password@localhost:3000"
 
+# Emulate login for local dev and testing
+export AUTH_EMULATION="true"
```

Comment:
for the sake of immediately obvious variable roles, can we rename this to `LOGIN_EMULATION`?